web/web.go

```diff
@@ -379,6 +379,12 @@ func (h *Handler) targets(w http.ResponseWriter, r *http.Request) {
 		tps[job] = append(tps[job], t)
 	}
 
+	for _, targets := range tps {
+		sort.Slice(targets, func(i, j int) bool {
```

Comment:
Hm yeah, just noticed. I wonder what our policy around requiring up-to-date versions of Go is. I mean, most users won't build Prometheus themselves, so it should be ok?

I could implement an entire sort interface, but that's kind of ugly if this nice new way exists now :)